code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +402,229 @@ components:
       description: |
         Ports may be specified as a list of ranges or single ports.
     QosProfile:
+      description: |
+        Data type with attributes of a QosProfile
+      type: object
+      properties:
+        name:
+          allOf:
+            - $ref: "#/components/schemas/QosProfileName"
+            - description: |
+                The name of the QoS profile.  This name is used to identify the profile when creating a session.
+                The name must be unique.
+        status:
+          allOf:
+            - $ref: "#/components/schemas/QosProfileStatusEnum"
+            - description: |
+              The status of the QoS profile.  The status is set to ACTIVE when the profile is created.
+              The status is set to INACTIVE when the profile is deleted.
+        targetMinUpstreamRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              This is the target minimum upstream rate for the QoS profile.
+
+              For 5G networks 3GPP Guaranteed Bit Rate (GBR) refers to a dedicated, fixed data rate assigned to
+              specific services, ensuring a minimum performance level. As per 3GPP TS 23.203,
+              GBR is a QoS parameter used to manage traffic classes in mobile networks. It
+              provides a stable data rate for latency-sensitive applications, such as voice calls or
+              video streaming, helping maintain a consistent user experience. When this attribute is set
+              this could imply that a GBR QCI is used, though mechanisms on the network can be used to
+              ensure a minimum performance level without using a GBR QCI.
+
+              The committed data rate allocated to specific services, ensuring a consistent level of
+              performance. For DOCSIS networks, the "Minimum Reserved Traffic Rate" is defined in the 
+              DOCSIS 3.1 - MAC and Upper Layer Protocols Interface Specification" 
+              and it ensures a consistent level of performance for specific services within the network.
+        maxUpstreamRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              The maximum best effort data
+        maxUpstreamBurstRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              The burst rate for best effort data
```

Comment:
I think it is not clear what is this in relation to `maxUpstreamRate`. If a provider designs a service to be "statistically" limit between `targetMinUpstreamRate` and  `maxUpstreamRate`, what does the burst rate represent.

Reply:
@jlurien This is an optional parameter without a default.  So if you service don't have a burst rate then it shouldn't define one.

Do we need update all of the optional values to define the QoS profile behaviour if that parameter isn't defined?

Reply:
Yes, I get that it's optional. Comment was to enhance the description of "burst rate", I feel it is not something easily understood without further explanation in this context.

Reply:
It should at least be clarified that this is NOT the 3GPP "Maximum Burst Size", which is measured in bytes, not bits/second. But I agree a proper definition would be better. As "burst rate" is not a 3GPP concept, I guess it comes from fixed networks?